README.md

```diff
@@ -60,7 +63,35 @@ tox
 
 ## Runtime Environment
 
-Install (from sources) system-wide:
+### Install pre-requirements
```

Comment:
IMHO it would be nice to also explain the deps a little bit for #256, especially why the dependencies are needed.

Reply:
Especially the word "introspection data" should be mentioned because this is what the error message in #256 says.

Reply:
Could you please recommend the text here?
Form my point of view, if someone is interested in understanding what each package is about, they can look it up.

Reply:
How about something like the following?

```
Despite the pure python dependencies (that can also be found in requirements.txt), Nmstate also needs NetworkManager to be running on the local system to be able to configure the local network state. To access NetworkManager Nmstate need libnm and the corresponding introspection data (`NM-1.0.typelib`, here provided by `python-gobject-base`). To manage OvS, Nmstate needs the packages `NetworkManager-ovs` and `openvswitch`. NetworkManager might need to be restarted after installing the OvS plugin and starting the openvswitch service.

Additionally, some bugs in NetworkManager require special configuration snippets.
```

Reply:
Done